python/ray/data/_internal/block_batching.py

```diff
@@ -148,7 +148,7 @@ def _format_batch(batch: Block, batch_format: str) -> BatchType:
     else:
         raise ValueError(
             f"The given batch format: {batch_format} "
-            f"is invalid. Supported batch type: {BatchType}"
+            f"is invalid. Supported batch type: {'default', 'pandas', 'pyarrow', 'numpy'}"
```

Comment:
The lint job is failing with this error:
```
python/ray/data/_internal/block_batching.py:151:89: E501 line too long (90 > 88 characters)
```

Could you wrap the line?

Reply:
It should be ok now.